### ISSUE TITLE
iOS VoiceOver: cannot access image settings including adding alt text or a caption

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/gutenberg-mobile#598]
When VoiceOver is turned on in the iOS Accessibility settings, it's not possible to access any of the edit details in an image block such as adding a caption, adding alt text, adding a link, or removing the image.

Steps to reproduce:

1. Create a new post with an image in it.
1. Go to Settings > General > Accessibility > VoiceOver and turn VoiceOver on.
1. Go back to WPiOS and open the post created in step 1.
1. Double tap to select an image block.
1. Try to add a caption.
1. Try to add alt text to the image (gear icon is unresponsive).
1. Try to add a link for the image (gear icon is unresponsive).
1. Try tapping the trash icon to remove the image.

<strong>Result:</strong> with VoiceOver turned on, I couldn't find a way to access the image settings such as caption, alt text, and link and I couldn't remove the image. ([2m6s](https://cloudup.com/cSV6Sl3mefP))

<sup>Tested with iPhone 6S, iOS 12.1.4, WP Internal 11.8.0.20190211.</sup>

cc @SiobhyB (internal reference: 1773488-zen)

**Comment by etoledom** [comment]
Fixed via https://github.com/wordpress-mobile/gutenberg-mobile/pull/794 and https://github.com/wordpress-mobile/gutenberg-mobile/pull/920
